src/SFML/Window/DRM/InputImplUDev.cpp

```diff
@@ -628,13 +629,9 @@ void InputImpl::setMousePosition(const Vector2i& position, const WindowBase& /*r
 ////////////////////////////////////////////////////////////
 bool InputImpl::isTouchDown(unsigned int finger)
 {
-    for (const auto& slot : touchSlots)
-    {
-        if (slot.id == static_cast<int>(finger))
-            return true;
-    }
-
-    return false;
+    return std::any_of(touchSlots.cbegin(),
+                       touchSlots.cend(),
+                       [finger](const TouchSlot& slot) { return slot.id == static_cast<int>(finger); });
```

Comment:
This is neat. Which check recommended this?

Reply:
`readability-use-anyofallof`